hydroflow/Cargo.toml

```diff
@@ -11,24 +11,12 @@ macros = [ "hydroflow_macro", "hydroflow_datalog" ]
 [[example]]
 name = "chat"
 
-[[example]]
-name = "covid_tracing"
-
-[[example]]
-name = "covid_tracing_dist"
```

Comment:
covid_tracing and covid_tracing_dist aren't deprecated - they use the core api

Reply:
I'd argue that's deprecated. We're not documenting it.